apidoc/Circle.yml

```diff
@@ -46,6 +46,11 @@ properties:
     type: Number
     default: 10
 
+  - name: opacity
+    summary: Opacity of this map circle, from 0.0 (transparent) to 1.0 (opaque).
+    type: Number
+    default: 1.0 (opaque)
```

Comment:
Can you put `since: "6.0.0"` in here before next Friday? Then I'll try to get it into 6.0.0